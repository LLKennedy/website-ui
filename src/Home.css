.Home {
  text-align: center;
}

.Home-header {
  background-image: url(/tiles-transparent.png);
  background-color: black;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  mix-blend-mode: screen;
  color: white;
  text-shadow: 0.4vmin 0.4vmin #090172;
}

.Home-link {
  color: rgb(146, 224, 255);
  mix-blend-mode: screen;
  text-shadow: 0.4vmin 0.4vmin #090172;
}