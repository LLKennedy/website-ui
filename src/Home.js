import React from 'react';

export default class Home extends React.Component {
    render() {
        return <div/>;
    }

    doStuff(x, y) {
        return x + y;
    }
}