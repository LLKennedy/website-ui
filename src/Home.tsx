import React from 'react';
import './Home.css'
import Detail from './Detail';

export default class Home extends React.Component {
    render() {
        return (
            <div className="Home">
                <header className="Home-header">
                    Hi, I'm Luke Kennedy.<br />
                I write code for fun and profit.<br />
                    <a href="https://www.github.com/LLKennedy" className="Home-link">Here's my Github!</a>
                </header>
                <Detail />
            </div>
        )
    }
}