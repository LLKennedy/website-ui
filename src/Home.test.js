import React from 'react';
import ReactDOM from 'react-dom';
import Home from './Home';

it('renders without crashing', async () => {
    const div = document.createElement('div');
    await ReactDOM.render(<Home />, div);
    await ReactDOM.unmountComponentAtNode(div);
});

it('can add things', async () => {
    const home = new Home();
    expect(home.doStuff(3,4)).toBe(7);
})