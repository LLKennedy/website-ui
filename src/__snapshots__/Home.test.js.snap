// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Home"
>
  <header
    className="Home-header"
  >
    Hi, I'm Luke Kennedy.
    <br />
    I write code for fun and profit.
    <br />
    <a
      href="https://www.github.com/LLKennedy"
    >
      Here's my Github!
    </a>
  </header>
</div>
`;
